kernel/config/definitions/kernel.yml

```diff
@@ -14,7 +14,7 @@
       - string:
           name: DISTROS
           description: "A list of distros to build for. Available options are: xenial, centos7, centos6, trusty, precise, wheezy, and jessie"
-          default: "centos7 precise xenial"
+          default: "centos7 xenial"
```

Comment:
Sure, I've added trusty. I've removed precise from defaults just because there does not seem to be any nodes that would be providing it at the moment -- at least I did not get any in ~15 minutes or so before I cancelled the job.